packages/design-system/src/components/combobox.tsx

```diff
@@ -95,10 +95,14 @@ export const ComboboxListboxItem = forwardRef(ListboxItemBase);
 
 export const Combobox = Popper;
 
+export const StyledPopperContent = styled(PopperContent, {
+  zIndex: theme.zIndices.max,
```

Comment:
It is already rendered inside of a portal, why do we need zIndex here?  I am afraid this will end up with another problem in the end, e.g. if a popover needs to be on top

Reply:
Portal does not change depth. Everything with zindex overlaps it.

Reply:
right, but we usually dont' set z-index on portals, the last one just wins, so they manage themselves automatically? did we end up having another portal with z-index?

Reply:
I added position: sticky for style source section which is overlapped by css inputs.

Reply:
then it should be just locally set, not globally in design system